### PR TITLE
Maid clothes improvements 2

### PR DESCRIPTION
**Pull request by Termineitor244**
#### Summary
Content "Adjust some values of the maid clothes and adds a new maid dress"

#### Purpose of change
#52971 had some problems so I started this PR to try again:
I was passing over the items and they seemed a little weird in some values, so I decided to adjust them while I was there. When I was modifying the maid clothes and searching for models of the clothing, I realized that the current maid dress is a "functional" maid dress, meaning, a normal dress that would use someone cleaning... But most maid dresses over there are just for cosplay, so I decided to introduce a new one. 

#### Describe the solution
Adjust the values of the maid dress, hat and long white gloves, as well as creating a new, shorter maid dress.

#### Describe alternatives you've considered
None.

#### Testing
Started a new game, searched for the new item, it spawns correctly and the other items are still there too.

#### Additional context
Who decided to implement the maid dress in the game as a blue dress?

[This](https://www.aliexpress.com/item/New-Women-Vintage-French-Maid-Long-Dress-Costume-Cosplay-Stage-Fancy-Dress-Plus-size-House-Keeper/32869838224.html) is how its supposed to be the original maid dress, the new one would be... Maybe something like [this](https://www.pinterest.com/pin/393502086165519236/), there are a lot of similar models on the internet of these types of maid clothes.

### CODE REVIEW

**Comment by Termineitor244** [comment]
> monster_attack_test.cpp:230: FAILED

I don't think its related to this PR.

File: data/json/items/armor/gloves.json [hunk 1]
```diff
@@ -665,14 +665,19 @@
...
     "symbol": "[",
     "looks_like": "gloves_light",
-    "color": "light_blue",
+    "color": "white",
     "armor": [
       { "covers": [ "hand_l", "hand_r" ], "coverage": 100, "encumbrance": 2 },
-      { "covers": [ "arm_l", "arm_r" ], "coverage": 50, "encumbrance": 2 }
+      {
+        "covers": [ "arm_l", "arm_r" ],
+        "specifically_covers": [ "arm_upper_l", "arm_upper_r", "arm_elbow_l", "arm_elbow_r", "arm_lower_l", "arm_lower_r" ],
```
Comment (bombasticSlacks) [inline data/json/items/armor/gloves.json:673]:
```suggestion
        "specifically_covers": [ "arm_elbow_l", "arm_elbow_r", "arm_lower_l", "arm_lower_r" ],
```

File: data/json/items/armor/gloves.json [hunk 2]
```diff
@@ -665,14 +665,19 @@
...
+        "coverage": 50,
```
Comment (bombasticSlacks) [inline data/json/items/armor/gloves.json:674]:
```suggestion
        "coverage": 100,
```

File: data/json/items/armor/gloves.json [hunk 3]
```diff
@@ -665,14 +665,19 @@
...
+        "encumbrance": 2
+      }
```
Comment (bombasticSlacks) [inline data/json/items/armor/gloves.json:676]:
```suggestion
      },
      {
        "covers": [ "arm_l", "arm_r" ],
        "specifically_covers": [ "arm_upper_l", "arm_upper_r" ],
        "coverage": 50,
      }
```

File: data/json/items/armor/torso_clothes.json [hunk 4]
```diff
@@ -525,6 +525,39 @@
...
+    "price_postapoc": 50,
+    "material": [ "cotton" ],
+    "symbol": "[",
+    "looks_like": "maid_dress",
+    "color": "dark_gray",
+    "armor": [
+      { "covers": [ "torso" ], "specifically_covers": [ "torso_upper", "torso_lower" ], "coverage": 65, "encumbrance": 4 },
+      {
+        "covers": [ "leg_l", "leg_r" ],
+        "specifically_covers": [ "leg_hip_l", "leg_hip_r", "leg_upper_l", "leg_upper_r" ],
```
Comment (bombasticSlacks) [inline data/json/items/armor/torso_clothes.json:546]:
```suggestion
        "specifically_covers": [ "leg_hip_l", "leg_hip_r" ],
```

File: data/json/items/armor/torso_clothes.json [hunk 5]
```diff
@@ -525,6 +525,39 @@
...
+        "coverage": 40,
```
Comment (bombasticSlacks) [inline data/json/items/armor/torso_clothes.json:547]:
```suggestion
        "coverage": 100,
```

File: data/json/items/armor/torso_clothes.json [hunk 6]
```diff
@@ -525,6 +525,39 @@
...
+        "encumbrance": 2
+      },
```
Comment (bombasticSlacks) [inline data/json/items/armor/torso_clothes.json:549]:
```suggestion
      },
      {
        "covers": [ "leg_l", "leg_r" ],
        "specifically_covers": [  "leg_upper_l", "leg_upper_r" ],
        "coverage": 50,
      },
```

**Review by bombasticSlacks — CHANGES_REQUESTED** [review]
something like this

**Pull request closed without merging**